src/main/views/lists/tabs/contactTypes/editContactType.njk

```diff
@@ -0,0 +1,71 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
```

Comment:
Is there a separate javascript file for this, or is it shared by the above one?

Reply:
shared javascript